### PR TITLE
Feat/periodic aggregation

### PR DESCRIPTION
**Pull request by jantischhoefer**
### Summary
Removed the database implementation from project-daedalus2.

### Details
Moved the currently working database implementation back to telestion-core and updated config.json

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

**Comment by jantischhoefer** [comment]
We can merge this PR to have a clean working database implementation, but now again back in telestion-core.
I fixed the imports and changed the config.json.
If the PR in telestion-core is merged and daedalus2 uses the new telestion-core, everything should be working as before.

**Review by fussel178 — CHANGES_REQUESTED** [review]
The indentation is all over the place. :see_no_evil: 
I think, we need some sort of an indentation check for the application site of the project...

For now, can you please indent your modified in a persistent way?

**Comment by fussel178** [comment]
And should we resolve the merge conflicts together in the near future? ^^

**Comment by jantischhoefer** [comment]
> The indentation is all over the place. 
> I think, we need some sort of an indentation check for the application site of the project...
> 
> For now, can you please indent your modified files in a persistent way?

I don't know where that indentation is coming from, because in my editor everything is looking fine.
Is there any settings issue on my side?

**Review by jantischhoefer — COMMENTED** [review]
io.vertx:vertx-mongo-client can be removed from build.gradle

Indentation will be fixed with spaces instead of tabs?

**Review by jantischhoefer — COMMENTED** [review]
SystemTPublisher can be uncommented, once telestion-core merged the feat-database-new branch.

**Review by jantischhoefer — COMMENTED** [review]
Non impact change to SimpleTest.java

**Comment by jantischhoefer** [comment]
@fussel178 i reviewed the requested changes and tried to fix the indentation. Please see, if the indentation is now correct. Seemed to be a problem with my editor conflicting with the projects .editorconfig

I also resolved the conflicts, because the only conflicts were the correction of the imports for our database verticles, which should be in telestion-core.

**Comment by pklaschka** [comment]
DO NOT MERGE:
Blocked until https://github.com/wuespace/telestion-core/pull/273 is both merged **and released**.

File: application/conf/config.json
Comment (fussel178) [inline application/conf/config.json:11]: Here the "default" is 2 spaced indented.
And your line is 1 tab and 2 spaces indented.
I think, it's reasonable to stick to 2 spaces indentation for this file.

**Review by fussel178 — CHANGES_REQUESTED** [review]
Some more stylistic changes. Sorry :see_no_evil:

**Comment by fussel178** [comment]
Closing, because code now resides in https://github.com/wuespace/telestion-extension-mongodb.

**Pull request closed without merging**
